backend/common/src/main/java/ai/verta/modeldb/common/CommonJdbiUtil.java

```diff
@@ -0,0 +1,324 @@
+package ai.verta.modeldb.common;
+
+import ai.verta.modeldb.common.config.DatabaseConfig;
+import ai.verta.modeldb.common.config.RdbConfig;
+import ai.verta.modeldb.common.exceptions.UnavailableException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Calendar;
+import java.util.Properties;
+import liquibase.Contexts;
+import liquibase.LabelExpression;
+import liquibase.Liquibase;
+import liquibase.configuration.GlobalConfiguration;
+import liquibase.configuration.LiquibaseConfiguration;
+import liquibase.database.DatabaseFactory;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.DatabaseException;
+import liquibase.exception.LiquibaseException;
+import liquibase.exception.LockException;
+import liquibase.lockservice.LockServiceFactory;
+import liquibase.resource.FileSystemResourceAccessor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public abstract class CommonJdbiUtil {
```

Comment:
```suggestion
public abstract class CommonDBUtil {
```
This doesn't seem specific to the database method (jdbi vs hibernate), right?

Reply:
Renamed